src/react-app-env.d.ts

```diff
@@ -1,12 +1 @@
 /// <reference types="react-scripts" />
-
-declare module "react-eventbrite-popup-checkout" {
-  interface PropType {
-    ebEventId: string;
-    className: string;
-    isModal?: boolean;
-    onOrderComplete?: () => void;
-  }
-  const EventbriteButton: React.FunctionComponent<PropType>;
-  export default EventbriteButton;
-}
```

Comment:
[note] This relates to the removed dependency, which didn't appear to be used anymore.